### DISCUSSION TITLE
MACOS Package not working?

### DISCUSSION THREAD

**Discussion by theerudirefrog** [monocasual/giada#659] [General]
giada-v0.25.0-x86_64-macos

is giving me a Damaged message.
Any thoughts?

(Mac Mini M1 MacOS 13.4.1 (c) )

**Comment by gvnnz** [comment]
Hey @theerudirefrog, yes the mac package is still unsigned, which makes the OS unhappy. Please follow the instructions [at the bottom of this page](https://giadamusic.com/documentation-introduction) for now. Thanks!
